src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -195,6 +195,16 @@ trait ClusterComponent extends LeoComponent {
         }
     }
 
+    def listActiveClusterByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
```

Comment:
Usually `list` methods return a `Seq`, I would name this with `get`.

Actually looking below, it seems like `getActiveClusterForDnsCache` already does what we want. Maybe rename that method to `getActiveClusterByNameMinimal` and use that in `LeonardoService.initiateClusterCreation`.